projects/packages/videopress/src/client/lib/get-chapter/index.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * internal dependencies
+ */
+import { GetChapterArgs } from './types';
+
+/**
+ *
+ * @param {GetChapterArgs} args - fetch chapter arguments.
+ * @returns {Promise} The chapter.
+ */
+
+const getChapter = async ( {
+	guid,
+	token,
+	isPrivate,
+	chapter,
+}: GetChapterArgs ): Promise< string | null > => {
+	if ( ! chapter?.src ) {
+		return;
+	}
+	if ( isPrivate && ! token ) {
```

Comment:
In the previous code this line had the following comment, should we add it back?
```
// Wait for the token when the video is private.
```